react/ButtonGroup/ButtonGroup.js

```diff
@@ -0,0 +1,17 @@
+import styles from './ButtonGroup.less';
+import React, { PropTypes } from 'react';
+
+export default function ButtonGroup({ children }) {
+  return (
+    <div>{children.map(child => React.cloneElement(child, { className: styles.button }))}</div>
```

Comment:
Not right now, but it increases the likelihood of a bug creeping in as we iterate on this component.